tool/gravity/cli/clusterupdate.go

```diff
@@ -196,6 +197,12 @@ const gravityResumeServiceName = "gravity-resume.service"
 // executeOrForkPhase either directly executes the specified operation phase,
 // or launches a one-shot systemd service that executes it in the background.
 func executeOrForkPhase(env *localenv.LocalEnvironment, updater updater, params PhaseParams, operation ops.SiteOperation) error {
+	if params.isResume() {
+		if err := verifyAgentsActive(env); err != nil {
```

Comment:
I would try to attempt to start agents automatically and only report errors if this is not possible - less friction for a user.

Reply:
Good suggestion :+1:. Updated in c1b4e6b